src/components/Root/Root.tsx

```diff
@@ -26,16 +26,15 @@ const Root: FC = () => {
 
   const [searchParams, setSearchParams] = useSearchParams();
 
-  const configSource = useMemo(() => getConfigSource(searchParams, settingsQuery.data), [searchParams, settingsQuery.data]);
-
+  const { configSource, env } = useMemo(() => getConfigSource(searchParams, settingsQuery.data), [searchParams, settingsQuery.data]);
```

Comment:
Can you think of another term for these besides environments and the abbreviation env?  It will be easy to confuse this with the .env file and values.  Maybe 'host', 'context', 'platform', or even just 'backend'?  Or we could make the variable really verbose but easier to read, like 'deploymentEnvironment'?

Reply:
@dbudzins jwApiHost / jwApiHostVersion?

Reply:
apiEnvironment?